keras_nlp/layers/transformer_decoder.py

```diff
@@ -134,73 +134,59 @@ def _build(self, input_shape, has_cross_attention):
         self._input_shape = input_shape
         self._has_cross_attention = has_cross_attention
         feature_size = input_shape[-1]
-        self._attention_head_size = int(feature_size // self.num_heads)
-        self._self_attention_layer = keras.layers.MultiHeadAttention(
+        head_size = int(feature_size // self.num_heads)
+
+        # Self attention layers.
+        self._self_attn_layer = keras.layers.MultiHeadAttention(
```

Comment:
The Keras style guide discourages abbreviations and actually uses "attention" as an example ([doc]( https://github.com/keras-team/governance/blob/master/keras_api_design_guidelines.md#seek-to-minimize-cognitive-load-for-our-users)).

Reply:
Yeah, sounds good.

It's not actually the long names I was disliking, it's how the long names and `black` interact to space things out.

But I think I can rework this in a way that's still well spaced and readable. Will push an update shortly. Probably will switch to explicit named args for MHA too.